File: wfsconnector/src/main/java/at/fhtw/leiwi/wfsconnector/GeoFeature.java
package at.fhtw.leiwi.wfsconnector;

import com.vividsolutions.jts.geom.Geometry;

/**
 * Created by Christoph on 01.03.2015.
 */
public class GeoFeature {
    private String type;
    private Geometry polygon;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public Geometry getPolygon() {
        return polygon;
    }

    public void setPolygon(Geometry polygon) {
        this.polygon = polygon;
    }
}

File: app/model/GeoFeature.scala
package model

import java.util.UUID

import com.vividsolutions.jts.geom.Geometry
import org.joda.time.{DateTime, LocalDateTime}

/**
 * Created by Christoph on 26.05.2015.
 */
case class GeoFeature(id: UUID, description: String, value: Double, coordinate: Geometry, measureDate :LocalDateTime, updateTime : DateTime)

object GeoFeatureTyp {
  val LuftgueteO3 = "Luftgüte O3"
}
